starters/README.md

```diff
@@ -0,0 +1,43 @@
+This is a collection of preconfigured ready-to-run applications which are useful for
+the development and testing of library components.
+
+For complete integration examples have a look at the [Miranum Consulting Repository](https://github.com/FlowSquad/miranum-consulting).
+
+## Spring Boot Starters
+
+### spring-examples
+
+This directory contains implementation examples that extent miranum functionality for Spring.
+
+### spring-web-c7
+
+This starter is pre-configured with the following components:
+
+* Camunda-7 adapter
+* Message adapter for camunda-7
+* CloudEvents adapter for spring-web
+* CloudEvents adapter for messages
+
+The started Spring Boot Web Application will be available at `localhost:5050`.
+
+Test to send a message via cURL:
```

Comment:
⛏️ - I would recommend specifying that the process needs to be started first of all. 
💭 - Maybe it makes more sense to have a process with a message start event and a user-task for the purpose of this readme. Every time you send a message via cURL a new process instance is started. 

What do you think?

Reply:
Yes, my first implementation was a message start event but the current implementation of the c7 adapter expects a correlation key and puts it hardcoded into a map under the key "correlationKey". I didn't manage to get the process started this way and thought let's get this merged first because this could be an optimization.

Reply:
That's great feedback. We should raise an Issue for this. It should be possible to trigger a message start event in C7. 
It would be nice though if we can add a sentence to the readme so a potential user get reminded to start a process instance.